travis_slack_notif.sh

```diff
@@ -0,0 +1,9 @@
+travis_job_id=$1
```

Comment:
Little out of scope but how to re-use this script for CI plugin configuration?

Reply:
We can make this script a bit more generic by passing variables for slack channel and plugin name. And maybe by move this script into the TriggerCi repository.